docs/visual-testing/workflows/review.md

```diff
@@ -124,6 +124,24 @@ There is also an option to reject the snapshot. This will mark the snapshot as "
 
 More information about reviewing and approving diffs can be found in [Subsequent Test Execution Review](#subsequent-test-execution-review).
 
+##### Diff inspection
+
+The Diff Inspection Tool is a valuable utility that helps identify and verify which DOM changes have impacted the UI. It offers a visual representation of the differences between two versions, making it easy to spot modified, added, or removed elements.
```

Comment:
The DOM Inspection tool....

(We call it DOM inspector in UI, I believe, so we can be consistent here too)

Reply:
Also, this last sentence can emphasize that we mean code changes:
...making it easy to spot modified, added, or removed elements in the code.

Reply:
Also, maybe one more sentence can be added after this one to bluntly explain how people can interact with the feature. Something like:

With this option enabled, you will be able to click on or cycle through any change found on a snapshot (indicated in pink color), and observe DOM changes for it in the panel on the right.